index.html

```diff
@@ -9,5 +9,12 @@
   </head>
   <body>
     <header>QuizApp for WebDevs</header>
+    <main>
+      <section class="card1">
+        <p class="question">What property flips the axes in flexbox?</p>
+        <button class="answer">Show answer</button>
+        <button class="bookmark"></button>
```

Comment:
The class names are not specific enough: they do not tell that the HTML element is a button. Consider using `button-answer` and `button-bookmark`.